.changes/next-release/api-change-imagebuilder-62729.json

```diff
@@ -0,0 +1,5 @@
+{
+  "type": "api-change",
+  "category": "``imagebuilder``",
+  "description": "Add all ``imagebuilder`` modeled paginators that we don't block."
```

Comment:
I'd remove "we don't block" from here. A CLI user reading the changelog may not know who "we" is or what it means to "block".